scripts/zones/Sacrarium/IDs.lua

```diff
@@ -46,6 +46,7 @@ zones[xi.zone.SACRARIUM] =
             { 16892074, { [xi.race.GALKA]    = true } },                            -- Indich (galka)
         },
         OLD_PROFESSOR_MARISELLE = 16891970,
+        ELEL                    = 16891948,
```

Comment:
we have that handy get-by-name stuff now so that we don't need the static ID.

Reply:
done